src/assets/md/FAQ.md

```diff
@@ -14,28 +14,18 @@ Through this method of order matching, liquidity is shared amongst all assets tr
 
 The Gnosis Protocol smart contracts have been audited externally. Check this section on their [DevPortal](https://dfusion-docs.dev.gnosisdev.com/docs/devguide04/) for further information, including the audit report and the code repository.
 
-**How do I trade on Mesa?**
-
-The first thing you have to do is enable the token(s) you would like to trade on the `Balances` page. Next, from the current `Balances` page, you should click on `+`, and there you will have to deposit this token into your Exchange Wallet. If you intend on selling ETH, you need to [wrap it first](https://weth.io/) (Wrapping ETH means converting it into WETH, so that it is technically compatible with the ERC-20 technical token standard.).
-
-After enabling and depositing the tokens you would like to trade, click on the `Order` page, select the tokens you would to trade, set the limit price and expiration time, and then submit the order.
-
-As with normal limit orders, it is important to note that if the batch clears at a better price (for the opposite side of the trade) than that of your order, your order will not be matched. If there is not enough volume, your order may also be partially filled.
-
-**Can I create a trade without funding?**
+**Can I create a trade without depositing to the smart contract?**
```

Comment:
I suggest "Can I create an order without depositing into my exchange balance first?" or "Can I create an order without funding on the exchange"?
1. they are not trading for sure  - so using trading instead of order is confusing. 
2. we never speak about smart contracts. maybe Gnosis Protocol
3. ALSO this is possible if there is no deposit but they receive funds through a trade.